misc-tools/dj_make_chroot.in

```diff
@@ -411,9 +386,7 @@ echo "This file should not be readable inside the judging environment!" \
     > "$CHROOTDIR/etc/root-permission-test.txt"
 chmod 0640 "$CHROOTDIR/etc/root-permission-test.txt"
 
-umount "$CHROOTDIR/dev/pts"
-umount "$CHROOTDIR/sys"
-umount "$CHROOTDIR/proc"
+normal_cleanup
```

Comment:
I think it would be good to change the name to make it clear that this undoes the mounting, so e.g. `unmount_filesystems`.